### ISSUE TITLE
Feature Request: Allow pre-releases, e.g., alpha -> beta -> rc -> stable

### ISSUE THREAD

**Issue by christian-kreuzberger-dtx** [WyriHaximus/github-action-next-semvers#15]
Hi!

I love this Action, it helps me so much :+1:

One request: I just released a software with version "0.8.0-alpha".
IMO the next version/release would be "0.8.0-beta", or even just "0.8.0" (I hope you see where I am heading too), but certainly **not** 0.8.1 (which it is right now)

So it would be nice if we could get one level deeper and also allow to handle pre-releases like this here:
https://github.com/WyriHaximus/github-action-next-semvers/blob/93321afe5bfbe63b1c860d9fc8f2e8b73e636a13/src/Next.php#L35-L43

Example Logic (based on https://github.com/nikolaposa/version ):

```
if (len($version->getPreRelease()) != 0) {
    $output .= '::set-output name=patch::' . $version . PHP_EOL;
}
```
This would print 0.8.0 as the next version, if we just released 0.8.0-alpha

**Comment by WyriHaximus** [comment]
Hey @christian-kreuzberger-dtx!

Always good to hear that :+1: .

If you want to feel free to create the PR adding those features. Sound very useful to me.

**Comment by christian-kreuzberger-dtx** [comment]
I'll take a look at it!

**Comment by christian-kreuzberger-dtx** [comment]
Done, but it looks like there is a problem with composer.

**Comment by WyriHaximus** [comment]
@christian-kreuzberger-dtx cool thanks! Have a good guess what the composer issue is, will fix it and then you can rebase the PR 👍

**Comment by WyriHaximus** [comment]
@christian-kreuzberger-dtx This has now been released as [`v1.1.0`](https://github.com/WyriHaximus/github-action-next-semvers/releases/tag/v1.1.0)

**Comment by christian-kreuzberger-dtx** [comment]
Nice, I'll try it out!
